base/build.gradle

```diff
@@ -40,6 +40,8 @@ android {
         abortOnError false
         lintConfig file('lint.xml')
     }
+
+    //preBuild.dependsOn 'checkstyle'
```

Comment:
you don't want to depend on `preBuild` task. If you do that it will run on every build and it will slow down your development process. What you should do is to depend on `check` task. That is also the task that is run on CI.

Same goes for other modules.